src/key_command/command.rs

```diff
@@ -52,6 +52,7 @@ pub enum Command {
 
     SearchGlob(String),
     SearchString(String),
+    SearchInc(String),
```

Comment:
I think we can call this `SearchIncrement`, `SearchIncremental`, or `IncrementalSearch`.
More descriptive and the user won't see it anyways.

Reply:
Done!